openpype/hosts/maya/plugins/publish/validate_maya_units.py

```diff
@@ -37,26 +45,44 @@ def process(self, context):
         self.log.info('Units (angular): {0}'.format(angularunits))
         self.log.info('Units (time): {0} FPS'.format(fps))
 
-        # Check if units are correct
-        assert linearunits and linearunits == 'cm', ("Scene linear units must "
-                                                     "be centimeters")
+        valid = True
 
-        assert angularunits and angularunits == 'deg', ("Scene angular units "
-                                                        "must be degrees")
-        assert fps and fps == asset_fps, "Scene must be {} FPS"\
-                                         "(now is {})".format(asset_fps, fps)
+        # Check if units are correct
+        if (
+            self.validate_linear_units
+            and linearunits
+            and linearunits != self.linear_units
+        ):
+            self.log.error("Scene linear units must be centimeters")
```

Comment:
Error message is incorrect - should be formatted with `self.linear_units`

Reply:
sure, that's why it is still draft. Putting kids to bed isn't compatible with finishing pull requests :) But when they are old enough, I'll let them do it :)